h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -135,6 +135,7 @@ protected Frame predictScoreImpl(Frame fr, Frame adaptFrm, String destination_ke
       
       StackedEnsemble.addModelPredictionsToLevelOneFrame(base, basePreds, levelOneFrame);
       DKV.remove(basePreds._key); //Cleanup
+      Frame.deleteTempFrameAndItsNonSharedVecs(basePreds, levelOneFrame);
```

Comment:
this omission is not due to refactoring nor blending, it was just not tested correctly: discovered that we were leaking keys when scoring SEs only with the AutoML test...
same below